cmd/bosun/web/static/partials/config.html

```diff
@@ -33,18 +32,90 @@
 	color: lightgray;
 }
 .lines .lineerror {
-	color: #a94442;
-	background: #f2dede;
+	color: black;
+	font-weight: bold;
+	background-color: lightgray;
 }
+
 </style>
+<label style="float:left; margin-right:15px;">Jump to:</label>
+<div class='row'>
+<div class="dropdown" ng-repeat="(type,list) in items" style="float:left; margin-right:5px;margin-bottom:15px;">
+  <button class="btn btn-primary dropdown-toggle" type="button" id="itemSelector" data-toggle="dropdown" aria-expanded="true">
+ 	{{type}}
+    <span class="caret"></span>
+  </button>
+  <ul class="dropdown-menu" role="menu" aria-labelledby="itemSelector">
```

Comment:
what are our thoughts about aria? we don't do it anywhere else. should we just start slowly, or do it nowhere until we decide?